lib/loader/index.js

```diff
@@ -135,6 +186,16 @@ module.exports = {
       var firstScript = getFirstScript();
 
       firstScript.parentNode.insertBefore(script, firstScript);
+
+      _loadedUrls[url] = true;
+
+      if (options.namespaceName) {
+        namespace.loadedUrls = _loadedUrls;
```

Comment:
Okay. As long as we're not advocating for namespace properties to be shared, this seems fine. If/when we do want apps to share state, we will want to use namespace somehow, won't we?